OrbitGl/App.cpp

```diff
@@ -1049,6 +1049,29 @@ std::shared_ptr<Process> OrbitApp::FindProcessByPid(uint32_t pid) {
 
 void OrbitApp::UpdateProcess(const std::shared_ptr<Process>& process) {
   absl::MutexLock lock(&process_map_mutex_);
+  // We want to move function map to new process object
+  auto it = process_map_.find(process->GetID());
+  if (it != process_map_.end()) {
+    // TODO: all the following needs to be addressed by
+    // separating client-only part of information to
+    // separate structures.
+    std::shared_ptr<Process> old_process = it->second;
+    std::map<uint64_t, std::shared_ptr<Module>>& old_modules =
+		old_process->GetModules();
+    std::map<uint64_t, std::shared_ptr<Module>>& new_modules =
+        process->GetModules();
+
+	// Move 'loaded' flag to new modules if possible.
+	for (auto module_entry : old_modules) {
+      uint64_t key = module_entry.first;
+      auto it = new_modules.find(key);
```

Comment:
This `it` shadows the `it` in the outer scope, maybe rename both to more descriptive names.

Reply:
done